rfc/4/README.md

```diff
@@ -30,8 +30,8 @@ Key | Value | Description
 `m0conf/profiles/<profile_fid>` | `{ "name": "<profile_name>", "pools": [ "<pool_name>" ] }` | "pools" - names of the _SNS_ pools associated with this profile.  `<profile_name>` and `<pool_name>`s are specified in the CDF.
 `m0conf/sites/<site_fid>` | `{ "state": "<HA state>" }` | [HA state](#ha-state) of this site.
 `m0conf/sites/<site_fid>/racks/<rack_fid>` | `{ "state": "<HA state>" }` | [HA state](#ha-state) of this rack.
-`m0conf/sites/<site_fid>/racks/<rack_fid>/encls/<encl_fid>` | `{ "state": "<HA state>" }` | [HA state](#ha-state) of this enclosure.
-`m0conf/sites/<site_fid>/racks/<rack_fid>/encls/<encl_fid>/ctrls/<ctrl_fid>` | `{ "node": "<node_fid>", "state": "<HA state>" }` | Fid of the corresponding node and [HA state](#ha-state) of this controller.
+`m0conf/sites/<site_fid>/racks/<rack_fid>/encls/<encl_fid>` | `{ "node": "<node_fid>", "state": "<HA state>" }` | Fid of the corresponding node and [HA state](#ha-state) of this enclosure.
```

Comment:
Suggestion: 
Use key `m0conf/sites/<site_fid>/racks/<rack_fid>/encls/<encl_fid>/nodes` instead. Reason: controller information is put under `encs/<encl_fid>/ctrls`. So it would look strange if we put node information directly at `encls` key.

BTW are these changes supported in hax source code? I guess it is hax who manages the data in these keys.

Reply:
Earlier, node information was added directly to `ctrls` key,  
as mentioned below

> m0conf/sites/<site_fid>/racks/<rack_fid>/encls/<encl_fid>/ctrls/<ctrl_fid> | { "node": "<node_fid>", "state": "HA state" } | Fid of the corresponding node and HA state of this controller.
> -- | -- | --

So now I have just moved the node info from `ctrls` key to `encls` key. Even @mssawant has described the same in the ticket. 

The suggestion looks good. Should we implement it? @mssawant , @knekrasov

Reply:
Why? we are mapping an enclosure to node, so as earlier we mapped a controller to node we will now map an enclosure to node. But also have controllers as
`m0conf/sites/<site_fid>/racks/<rack_fid>/encls/<encl_fid>/ctrls/<ctrl_fid>/drives/<drive_fid>` | `{"sdev":"<sdev_fid>", "state":"<drive_state>"}`